src/Aspire.Hosting.Sdk/Aspire.RuntimeIdentifier.Tool/Aspire.RuntimeIdentifier.Tool.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Nesting .csprojs under folders that already contain a .csproj is typically not a good idea.

Reply:
Agreed, though in this case the parent project is a `NoTargets` SDK project meaning it doesn't compile anything, and the child project doesn't have any targets that could be packed by the parent by accident either. Main reason why I made it a child project, is that the parent is the one that packs this. I could change it of course, but this "feels" better. There is also precedent in dotnet/runtime where source generator projects are nested under the package that packs them.

Reply:
For example: https://github.com/dotnet/runtime/tree/main/src/libraries/System.Text.Json/gen

Reply:
in dotnet/runtime, it is the `src` project that packs them, which is a sibling, not in a parent directory.

Reply:
Oh -- fair enough. I don't feel strongly about this but I can change it if you want me to.